lib/stream/filter_slim_graph.js

```diff
@@ -0,0 +1,45 @@
+var keys = require("lodash/keys");
+var omit = require("lodash/omit");
+var through = require("through2");
+var assign = require("lodash/assign");
+var includes = require("lodash/includes");
+
+module.exports = function() {
+	return through.obj(function(data, enc, done) {
+		try {
+			done(null, filterGraph(data));
+		} catch (err) {
+			done(err);
+		}
+	});
+};
+
+var blackList = [
+	"npm",
+	"npm-convert",
+	"npm-crawl",
+	"npm-extension",
+	"npm-load",
+	"npm-utils",
+	"semver",
+	"@dev"
+];
```

Comment:
rather than using a blacklist, could we get the `loader.configMain` which should be `package.json!npm` and filter all dependencies (and dependency's dependencies) of that?